srcpkgs/maxcso/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'maxcso'
+pkgname=maxcso
+version=1.11.0
+revision=1
+build_style=gnu-makefile
+makedepends="libuv-devel liblz4-devel zlib-devel"
+depends="libuv liblz4 zlib"
```

Comment:
These are automatically detected by xbps-src so they don't need to be manually added:

```
SONAME: libuv.so.1 <-> libuv>=1.0.0_1
   SONAME: liblz4.so.1 <-> liblz4>=1.8.2_1
   SONAME: libz.so.1 <-> zlib>=1.2.3_1
```

Reply:
Fixed.